README.md

```diff
@@ -65,6 +65,7 @@ module "eks-blueprints-kubernetes-addons" {
     enable_aws_for_fluentbit              = true
     enable_argocd                         = true
     enable_ingress_nginx                  = true
+    enable_appmesh                        = true
```

Comment:
For consistency, let's align with the name of the chart which is `appmesh-controller`. So this should be `enable_appmesh_controller` and the module dir should be `appmesh-controller` as opposed to `aws-app-mesh